frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-list/experiment-list.component.ts

```diff
@@ -170,6 +170,7 @@ export class ExperimentListComponent implements OnInit, OnDestroy, AfterViewInit
   }
 
   filterExperimentByChips(tagValue: FLAG_SEARCH_SORT_KEY, type: EXPERIMENT_SEARCH_KEY) {
+    this.searchValue = tagValue;
```

Comment:
You already have a subscription for this change so don't set the value directly here.